src/pages/Profile/Profile.jsx

```diff
@@ -10,13 +10,15 @@ import Header from './Header/Header.jsx'
 import Top from './Top/Top.jsx';
 import Queue from './Queue/Queue.jsx';
 import Modal from './Modal/Modal';
+import SubmitModal from './Modal/SubmitModal';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/OrangeMGE-ITP/SmartQ-ITP-Front/pullRequest?prid=8748018)
```suggestion
import SubmitModal from "./Modal/SubmitModal";
```